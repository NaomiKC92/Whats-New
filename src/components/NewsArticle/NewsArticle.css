* {
  box-sizing: border-box;
  margin: 0;
  padding: 0;
}

.news-article {
  /* border: 1px solid red; */
  height: 400px;
  width: 47%;
}

.article-img {
  height: 205px;
  width: 100%;
}

.article-summary,
.article-title {
  background-color: #FFF;
}

.article-summary {
  height: 180px;
}

.article-url {
  background-color: #B56AF1;
  height: 55px;
  color: #FFF;
}

a {
  color: #FFF;
  font-size: 150%;
}