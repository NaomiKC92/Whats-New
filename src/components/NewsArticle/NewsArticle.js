import React from 'react';
import './NewsArticle.css';

const NewsArticle = (props) => {
  return (
    <div className='news-article'>
      <img src={props.img} />
      <h2>{props.headline}</h2>
      <p>{props.description}</p>
      <section>{props.url}</section>
    </div>
  )
}

export default NewsArticle;