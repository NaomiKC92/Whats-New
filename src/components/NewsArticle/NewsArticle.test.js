import React from 'react';
import ReactDOM from 'react-dom';
import NewsArticle from './NewsArticle';
import { shallow } from 'enzyme';

describe('NewsArticle', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<NewsArticle />)
    expect(wrapper).toMatchSnapshot()
  })
})