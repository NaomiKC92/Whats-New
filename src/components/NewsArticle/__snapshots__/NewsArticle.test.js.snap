// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewsArticle should match the snapshot 1`] = `
<div
  className="news-article"
>
  <img
    alt="Related to specific news article"
    className="article-img"
  />
  <h2
    className="article-title"
  />
  <p
    className="article-summary"
  />
  <section
    className="article-url"
  />
</div>
`;
